assets/src/edit-story/elements/media/visibleImage.js

```diff
@@ -18,39 +18,16 @@
  * External dependencies
  */
 import styled from 'styled-components';
-import { useRef, useCallback, useState } from 'react';
-
-/**
- * Internal dependencies
- */
-import useAverageColor from './useAverageColor';
 
 const Image = styled.img`
   display: block;
-  ${({ isTooBright }) =>
-    isTooBright &&
-    `
-    /* Using filter rather than box-shadow to correctly follow
-     * outlines in semi-transparent images like gif and png.
-     */
-    filter: drop-shadow( 0 0 5px rgba(0, 0, 0, 0.5) );
-  `}
+  height: 28px;
+  width: 28px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
```

Comment:
This image is stretched/squeezed to fit the square box. It should probably have a `object-fit: cover` to make it look better (but of course cropped).